Bump to 0.24.3

Quentinchampenois opened the pull request:
#### Tasks

* Remove admin menu monkey patch 
* Load Pol.is in admin menu with admin engine initializer

Quentinchampenois committed:
Move admin menu in initializers

Quentinchampenois committed:
Bump to 0.24.3

paulinebessoles reviewed (changes requested):
Could you review and merge this PR before merging this one? https://github.com/OpenSourcePolitics/decidim-polis/pull/29
Thanks!

paulinebessoles committed:
Fix polis bug on assemblies (#29)

* Prepare bump for 0.24.3

* Fix polis bug on assemblies

## **Describe the bug**
[Pol.is](http://pol.is) only works in participatory processes for now.
This PR aims to solve the issue encountered on Assemblies when trying to use the Pol.is component. 

## **Screenshots**
![https://s3-us-west-2.amazonaws.com/secure.notion-static.com/4f5bdebc-078e-4661-a0df-12bda2e17c29/Capture_decran_2021-05-12_a_12.11.31.png](https://s3-us-west-2.amazonaws.com/secure.notion-static.com/4f5bdebc-078e-4661-a0df-12bda2e17c29/Capture_decran_2021-05-12_a_12.11.31.png)

* Add empty line

Co-authored-by: Armand Fardeau <armandfardeau@users.noreply.github.com>

Co-authored-by: quentinchampenois <26109239+Quentinchampenois@users.noreply.github.com>
Co-authored-by: Armand Fardeau <armandfardeau@users.noreply.github.com>
